django_outbox_pattern/decorators.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=protected-access
```

Comment:
question: What is the motivation to include this bypass? Not is possible to remove that?